src/tox_uv/_installer.py

```diff
@@ -114,7 +131,15 @@ def _install_list_of_deps(  # noqa: C901, PLR0912
             if isinstance(arg, Requirement):  # pragma: no branch
                 groups["req"].append(str(arg))  # pragma: no cover
             elif isinstance(arg, (WheelPackage, SdistPackage, EditablePackage)):
-                groups["req"].extend(str(i) for i in arg.deps)
+                for i in arg.deps:
```

Comment:
Now that you made it not a comprehension you should not use a single letter variable anymore.

Reply:
You did not address this.